packages/ui/src/components/va-tabs/VaTabs.vue

```diff
@@ -106,6 +107,7 @@ export default defineComponent({
     right: { type: Boolean, default: false },
     center: { type: Boolean, default: false },
     grow: { type: Boolean, default: false },
+    withPagination: { type: Boolean, default: true },
```

Comment:
Better rename it to `hide-pagination` prop, so it is `false` by default.

```vue
<va-tabs hide-pagination />
```
vs

```vue
<va-tabs :with-pagination="false" />
```